app/src/main/java/com/rokt/roktdemo/data/service/RoktDemoService.kt

```diff
@@ -0,0 +1,17 @@
+package com.rokt.roktdemo.data.service
+
+import com.rokt.roktdemo.model.AboutRokt
+import com.rokt.roktdemo.model.DemoLibrary
+import retrofit2.http.GET
+import retrofit2.http.Headers
+import retrofit2.http.POST
+
+interface RoktDemoService {
+    @Headers("accept: text/plain")
```

Comment:
Just checking, have you checked with `Accept: application/json` ?

Reply:
I hadn't checked, thanks, updated.